sycl/doc/extensions/experimental/sycl_ext_oneapi_complex.asciidoc

```diff
@@ -211,17 +221,125 @@ namespace sycl::ext::oneapi::experimental {
 } // namespace sycl::ext::oneapi::experimental
 ```
 
-=== Mathematical operations
+=== Marray Complex Class Specialization
+
+This proposal also introduces the specialization of the marray class to
```

Comment:
I'd prefer that we use `marray` and `complex` throughout here, to make it clear when we're talking about a class.

(This comment applies to the whole of this section.)